README.md

```diff
@@ -69,9 +69,22 @@ Or you can submit all in one transaction.
 
 `Oracle` is a wrapper object of `DNSSEC.sol` Oracle smart contract.
 
-- `knownProof(proof)` returns true if the given proof already exists in `Oracle`.
-- `submit(dnsresult, params)` sends all unproven proofs into DNSSEC Oracle as one transaction in a batch.
+- `submitAll(dnsresult, params)` sends all unproven proofs into DNSSEC Oracle as one transaction in a batch.
```

Comment:
Does it send all of them, or only the ones necessary to get the leaf accepted?

Reply:
What would be the best way to describe? ` ...sends all unmatched proofs from the leaf till matched node  into DNSSEC...`?

Reply:
Maybe "submits all required proofs into the DNSSEC oracle"?